src/enums/EmployeeLevel.java

```diff
@@ -27,4 +27,19 @@ public static EmployeeLevel getStatusNameFromIndex(int num) {
         return null;
     }
 
+    public static String getTranslatedStatusNameFromIndex(int num) {
+        for (EmployeeLevel value : values()) {
+            if (value.levelValue == 1) {
+                return "INICIANTE";
+            }
+            if (value.levelValue == 2) {
+                return "EXPERIENTE";
+            }
+            if (value.levelValue == 3) {
+                return "AVANÇADO";
+            }
+        }
+        return null;
+    }
```

Comment:
Esse código não tá funcionando como o esperado. È sempre bom testar pra ver se a funcionalidade está como desejada.

Essa função não está utilizando o parâmetro `num`. Está iterando por todos os valores do enum, e, se for igual a 1, retorna "INICIANTE". Por isso, no teste que fiz, essa função sempre retorna "INICIANTE" independente do valor `num` inserido. O mesmo vale pro método da EnrollmentStatus.

Além do mais, a minha sugestão é que seria melhor a função receber um valor do próprio enum, ao invés de int. Assim poderiam chamar, por exemplo, `EmployeeLevel.getTranslatedStatusNameFromIndex(EmployeeLevel.BEGINNER);`

Também acho que não deveria retornar a palavra toda em maiúscula. Pra ser printada para o usuário, acho que seria melhor retornar "Iniciante", "Experiente", etc.

Outra sugestão é que seria bom usar um switch nessa lógica, pelo menos faz mais sentido pra mim do que else ifs.

Reply:
Perfeito, Michel. Vou refazer e providenciar as correções.